apps/omg_child_chain/test/omg_child_chain/integration/fee_server_test.exs

```diff
@@ -66,8 +66,10 @@ defmodule OMG.ChildChain.Integration.FeeServerTest do
       assert {:ok, default_fees} == FeeServer.transaction_fees()
 
       # corrupt file, refresh, check fees did not change
-      overwrite_fee_file(file_name, "[not a json]")
-      assert refresh_fees() =~ ~r/\[error\].*Unable to update fees/
+      assert capture_log(fn ->
+               overwrite_fee_file(file_name, "[not a json]")
```

Comment:
For me it's impossible we get log event before calling `refresh_fees()` therefor consider move `overwrite_fee_file(...)` outside capture_log lambda. What do you thnik?

Reply:
forget about interval message :update_fee_spec :D 
1. overwrite_fee_file overwrites file
2. send_interval :update_fee_spec run next
3. refresh_fees()  have no logs becase file don't change